core/src/main/scala/org/apache/spark/sql/execution/ddl.scala

```diff
@@ -616,3 +616,41 @@ case class UnDeployCommand(alias: String) extends RunnableCommand {
     Nil
   }
 }
+
+case class PutIntoValuesColumnTable(identifier: TableIdentifier,
+    colNames: Option[String],
+    values: Seq[Expression])
+    extends RunnableCommand {
+
+  def convertTypes(value: String, struct: StructField): Any = struct.dataType match {
+    case BinaryType => value.toCharArray().map(ch => ch.toByte)
+    case ByteType => value.toByte
+    case BooleanType => value.toBoolean
+    case DoubleType => value.toDouble
+    case FloatType => value.toFloat
+    case ShortType => value.toShort
+    case DateType => value
+    case IntegerType => value.toInt
+    case LongType => value.toLong
+    case _ => value
+  }
+
+  override def run(sparkSession: SparkSession): Seq[Row] = {
+    val snc = new SnappySession(sparkSession.sparkContext)
```

Comment:
Instead of creating a new session again use the passed in sparkSession by casting it to SnappySession. Refer to CreateTableUsingCommand#run
```
val session = sparkSession.asInstanceOf[SnappySession]
```

Reply:
Incorporated above changes